src/include/distributed/worker_transaction.h

```diff
@@ -22,6 +22,7 @@
 typedef enum TargetWorkerSet
 {
 	WORKERS_WITH_METADATA,
+	WORKERS_WITH_METADATA_EXCLUDE_CURRENT_WORKER,
```

Comment:
I'm open for alternative names for this enum. It seems too long now.

Reply:
`OTHER_WORKERS_WITH_METADATA`  ?

Reply:
I suggested `OTHER_WORKERS_WITH_METADATA` but it has not come here.